app/src/main/java/com/androidapp/taskmaster/activities/TaskDetailsActivity.java

```diff
@@ -25,7 +25,7 @@ protected void onResume () {
         super.onResume();
 
         preferences = getSharedPreferences(getPackageName() + "_preferences", Context.MODE_PRIVATE);
-        String task = preferences.getString(MainActivity.PRODUCT_NAME_TAG, "No task");
+        String task = preferences.getString(MainActivity.TASK_NAME_TAG, "No task");
         ((TextView) findViewById(R.id.taskDetailsActivityInputName)).setText(task);
     }
 }
```

Comment:
Troubleshooting TaskDetailsActivity:

- In your onCreate() method, you need a way to process the incoming Intent. (Hint: start with `Intent intent = getIntent()`,  then get the info you attached to the intent in your TasksRecycleViewAdapter)
- Check my notes on TasksRecycleViewAdapter, then make sure you have a property in this class that can hold the taskTitle